lib/i18n/backend/base.rb

```diff
@@ -253,7 +253,7 @@ def load_yml(filename)
         # toplevel keys.
         def load_json(filename)
           begin
-            ::JSON.parse(File.read(filename))
+            ::JSON.parse(File.read(filename), symbolize_names: true, freeze: true)
```

Comment:
I couldn't find the `freeze` option mentioned in the docs. So I tried it out:

```
[10] pry(main)> obj = JSON.parse(%Q{{ "one": "two" }}, freeze: true)
=> {"one"=>"two"}
[11] pry(main)> obj["one"].frozen?
=> false
```

It seems like it might not be a supported option. Unsupported options appear to be ignored. Here's an example of me using my name as an option:

```ruby
obj = JSON.parse(%Q{{ "one": "two" }}, ryan: true)
=> {"one"=>"two"}
```

Reply:
It's because you JSON is too old:
```
>> JSON.parse('"foo"', freeze: true).frozen?
=> true
>> JSON::VERSION
=> "2.5.1"
```

Which is fine, the feature is best effort.